grafanaDashboard.json

```diff
@@ -111,8 +111,7 @@
       "targets": [
         {
           "datasource": {
-            "type": "prometheus",
-            "uid": "FwnjnVA7z"
+            "type": "prometheus"
```

Comment:
This makes it so that the dashboard doesn't have to be modified to pick up the correct Grafana data source